spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
But can we asserting this precondition globally ?  Then we no need to do it in each tests.

And even you want do it in tests, I think you should put this line in a up-layer before, because you are doing the same thing under every describe.